### ISSUE TITLE
bad symbol

### ISSUE THREAD

**Issue by kaldown** [wsdjeg/SpaceVim#801]
Run vim and see broken symbol.

- OS: debian 9
- vim version: 8.0

I've installed 
1) DejaVu Sans Mono for Powerline
2) nerd-fonts

no custom config

![2017-08-26--21 59 08_1049x38_scrot](https://user-images.githubusercontent.com/3754017/29744290-d526c07a-8aaa-11e7-82ed-bc0d2506c9e2.png)

**Comment by wsdjeg** [comment]
@kaldown I think your font has not been installed successfully, or your terminal config is wrong.

**Comment by kaldown** [comment]
it's just default xfce4-terminal with no any config.
Fonts were installed as one mentioned in documentation.

**Comment by wsdjeg** [comment]
you need to change the terminal config to select the font you just installed.

**Comment by jm33-m0** [comment]
i am using xfce4-terminal too, and facing the same bad symbol. guess i need to switch to other terminals

**Comment by wsdjeg** [comment]
@jm33-m0 you need change the terminal config instead

**Comment by jm33-m0** [comment]
@wsdjeg but i am sure the nerd font is selected, and other symbols can be correctly displayed. 
P.S. verified: changing terminals doesn't solve this issue, tilix gives the same result

**Comment by wsdjeg** [comment]
please use new issue

**Comment by jm33-m0** [comment]
@wsdjeg ok, i am opening one

**Comment by oblitum** [comment]
Hi, sorry the noise, but I just wish to report a similar experience, it seems identical, but looks like it ended up ignored in these threads. I'll add more information.

I recently just switched back from Alacritty to xfce4-terminal for sake or undercurls, which Alacritty miss.

I use nerd fonts for a long time, and had no issue on Alacritty. When switching to xfce4-terminal I've hit this issue, I nailed it to a simple test, Alacritty can both display nerd glyph for `nf-mdi-lightbulb` (`f834`) and `nf-mdi-lightbulb_on` (`fbe6`), but xfce4-terminal can only show the nerd glyph for `nf-mdi-lightbulb`, it doesn't show a bulb for `nf-mdi-lightbulb_on`, it instead shows some unicode I ignore about: ﯦ.

Both Alacritty and xfce4-terminal are relying on the same font, with same fallbacks (there's just a single `~/.config/fontconfig/fonts.conf` for both).
